### DISCUSSION TITLE
Amazon blocking linking?

### DISCUSSION THREAD

**Discussion by YenRug** [audiamus/BookLibConnect#114] [General]
Just tried to set up my initial profile, believe I've gone through all of the steps correctly, upon pasting the "failed" page's URL I get the message from BookLibConnect:

Device registration failed.

Suggestive that they've changed something in the output so that it's no longer usable?

`https://www.amazon.co.uk/ap/maplanding?captcha_verified=1&openid.assoc_handle=amzn_audible_ios_uk&openid.claimed_id=https%3A%2F%2Fwww.amazon.co.uk%2Fap%2Fid%2Famzn1.account.AHQ3PI2XKNKD72GMQKMFCGNUSANQ&openid.identity=https%3A%2F%2Fwww.amazon.co.uk%2Fap%2Fid%2Famzn1.account.AHQ3PI2XKNKD72GMQKMFCGNUSANQ&openid.mode=id_res&openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&openid.op_endpoint=https%3A%2F%2Fwww.amazon.co.uk%2Fap%2Fsignin&openid.response_nonce=2023-09-20T13%3A32%3A02Z5968978476856269720&openid.return_to=https%3A%2F%2Fwww.amazon.co.uk%2Fap%2Fmaplanding&openid.signed=assoc_handle%2Cclaimed_id%2Cidentity%2Cmode%2Cns%2Cop_endpoint%2Cresponse_nonce%2Creturn_to%2Cns.pape%2Cpape.auth_policies%2Cpape.auth_time%2Coa2.authorization_code%2Ccaptcha_verified%2Csigned&openid.ns.pape=http%3A%2F%2Fspecs.openid.net%2Fextensions%2Fpape%2F1.0&openid.pape.auth_policies=http%3A%2F%2Fschemas.openid.net%2Fpape%2Fpolicies%2F2007%2F06%2Fnone&openid.pape.auth_time=2023-09-20T13%3A32%3A01Z&openid.sig=ZNrHNY1nd6ULt3enMqcSlgErBbJxRapyakzldQJSsPg%3D&serial=&openid.oa2.authorization_code=RHBPNxAcDhaXsiuDbVAYANsR&openid.ns.oa2=http%3A%2F%2Fwww.amazon.com%2Fap%2Fext%2Foauth%2F2`

**Comment by audiamus** [comment]
Can you provide a log file, log level 4?

**Reply by YenRug** [reply]
If you can point me to something explaining how to provide that, I will, thought you were asking about something the program creates but took a quick look and I see nothing for that.

**Reply by audiamus** [reply]
See the manual, chapter 4.

**Reply by YenRug** [reply]
Fiixed it, just as I pasted in the logs I suddenly remembered that Amazon Prime Games Hub doesn't like connecting whilst my VPN is enabled, didn't think it would affect the normal website operations but it obviously does. Oddly, my VPN is still set to the same country as I live in and the account is registered for, but Amazon obviously recognises that it's a VPN being used.
